docker/Dockerfile.osmo-smsc

```diff
@@ -10,6 +10,15 @@ ADD ./ /tmp/osmo-smsc
 
 RUN make -C /tmp/osmo-smsc/ install-rdepends install && rm -rf /tmp/osmo-smsc
 
+# reuse the start skript
+RUN cp /usr/bin/pharo-vm-nox /usr/bin/pharo-vm-nox.bak && \
+    dpkg --purge pharo-vm-core && \
+    mv /usr/bin/pharo-vm-nox.bak /usr/bin/pharo-vm-nox
```

Comment:
ah fun. we could just pass --nodisplay and then use the "pharo" binary. But that is fine as well.